<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studywindow</class>
 <widget class="QWidget" name="studywindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="questionLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>541</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>question</string>
   </property>
  </widget>
  <widget class="QLabel" name="answerLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Answer:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkAnswerButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>300</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Check Answer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>470</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Return to Deck</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>340</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Question</string>
   </property>
  </widget>
  <widget class="QLabel" name="feedbackLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>400</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Feedback:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="answerInput">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>431</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
